src/dropdowns/dropdown-item.ts

```diff
@@ -0,0 +1,57 @@
+import {Directive, Input, HostBinding, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownItem]',
+  host: {
+    'tabindex': '0',
```

Comment:
@sadesyllas Usually dropdown items are focusable elements, like the `<a>` tags on lightning's demo. By placing the directive on the `<li>` and manually changing the `tabindex` has two downsides. 1) We lose the default style SLDS applies on the focused `<a>` and instead we get the default outline of the browser 2) In a focused dropdown item that inside has `< a href="url">` does nothing if you hit enter, and instead open's the url if you click with the mouse.